persistence/src/commonMain/kotlin/com/wire/kalium/persistence/model/AuthSessionEntity.kt

```diff
@@ -0,0 +1,39 @@
+package com.wire.kalium.persistence.model
+
+import com.wire.kalium.persistence.dao.QualifiedIDEntity
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class ServerConfigEntity(
+    @SerialName("id") val id: String,
+    @SerialName("links") val links: Links,
+    @SerialName("metadata") val metaData: MetaData
+) {
+    @Serializable
+    data class Links(
+        @SerialName("apiBaseUrl") val api: String,
+        @SerialName("accountsBaseUrl") val accounts: String,
+        @SerialName("webSocketBaseUrl") val webSocket: String,
+        @SerialName("blackListUrl") val blackList: String,
+        @SerialName("teamsUrl") val teams: String,
+        @SerialName("websiteUrl") val website: String,
+        @SerialName("title") val title: String
+    )
+
+    @Serializable
+    data class MetaData(
+        @SerialName("federation") val federation: Boolean,
+        @SerialName("commonApiVersion") val apiVersion: Int,
+        @SerialName("domain") val domain: String?
+    )
+}
+
+@Serializable
+data class AuthSessionEntity(
+    @SerialName("user_id") val userId: QualifiedIDEntity,
+    @SerialName("token_type") val tokenType: String,
+    @SerialName("access_token") val accessToken: String,
+    @SerialName("refresh_token") val refreshToken: String,
+    @SerialName("wire_server") val serverLinks: ServerConfigEntity.Links
```

Comment:
Maybe I'm missing something, but wouldn't be better to store `ServerConfigEntity.id` here instead of `ServerConfigEntity.Links`?

Reply:
Ok, I get it, but now Links data is duplicated in both auth session and server config, wouldn't be better to keep both in respective tables, we could even keep Links and MetaData in separate tables and just operate on IDs? 
I don't know it that's possible but in case of needing to store the server config before doing any operation, we could store it already only in its proper table and keep only ID of such row in auth session data 🤔

Reply:
I wanted to avoid having this connection between the ID (since it's something we added and not part of the server config) and the AuthSession